package.json

```diff
@@ -21,11 +22,13 @@
     "url": "https://github.com/Schamper/nodebb-plugin-poll/issues"
   },
   "dependencies": {
-    "async": "~1.5.2",
-    "xregexp": "~3.1.0",
```

Comment:
Mm, I see this was removed, where was it used, if at all?

Reply:
It wasn't used at all.